solarforecastarbiter/metrics/probabilistic.py

```diff
@@ -82,17 +85,15 @@ def brier_skill_score(fx, fx_prob, ref, ref_prob, obs):
         interval.
     ref_prob : (n,) array_like
```

Comment:
I guess this would also become `fx_ref_prob` for consistency with `fx_ref` (*if* we decide to do that).

Reply:
Yeah, that's the "trade-off"; `ref` and `ref_prob` or `fx_ref` and `fx_ref_prob`. I personally prefer `ref` and `ref_prob`, but I'm okay either way. So it's your call.